packages/nouns-contracts/contracts/governance/NounsDAOV3Proposals.sol

```diff
@@ -225,23 +225,27 @@ library NounsDAOV3Proposals {
         checkProposalTxs(txs);
         uint256 proposalId = ds.proposalCount = ds.proposalCount + 1;
 
-        (uint256 votes, address[] memory signers) = verifySignersCanBackThisProposalAndCountTheirVotes(
-            ds,
-            proposerSignatures,
-            txs,
-            description,
-            proposalId
-        );
-
         uint256 adjustedTotalSupply = ds.adjustedTotalSupply();
-        uint256 propThreshold = checkPropThreshold(ds, votes, adjustedTotalSupply);
+
+        uint256 propThreshold = proposalThreshold(ds, adjustedTotalSupply);
+
         NounsDAOStorageV3.Proposal storage newProposal = createNewProposal(
             ds,
             proposalId,
             propThreshold,
             adjustedTotalSupply,
             txs
         );
+
+        (uint256 votes, address[] memory signers) = verifySignersCanBackThisProposalAndCountTheirVotes(
+            ds,
+            proposerSignatures,
+            txs,
+            description,
+            proposalId
+        );
+        require(votes > propThreshold, 'NounsDAO::propose: proposer votes below proposal threshold');
```

Comment:
since we're no longer reusing the prop threshold check code, maybe better this becomes a custom error, to get less auditing noise in future?

Reply:
sure. in this case I'm changing the error in both places